tbdex/offering/offering.go

```diff
@@ -73,7 +89,7 @@ type OfferingPayoutMethod struct {
 //   - It takes the algorithm identifier of the hash function and data to digest as input and returns
 //   - the digest of the data.
 func (o Offering) Digest() ([]byte, error) {
```

Comment:
I think the digest functionality here could mostly be shared at the `tbdex` root, albeit I get we want it as a receiver method for each tbdex message or resource type, maybe the receiver could call the root digest function

just a thought, not necessary here

Reply:
I think @phoebe-lew has that WIP